x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/action_clone.tsx

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonEmpty } from '@elastic/eui';
+import React, { FC } from 'react';
+import { isEqual } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { DataFrameAnalyticsConfig, isOutlierAnalysis } from '../../../../common';
+import { isClassificationAnalysis, isRegressionAnalysis } from '../../../../common/analytics';
+import { CreateAnalyticsFormProps } from '../../hooks/use_create_analytics_form';
+import { State } from '../../hooks/use_create_analytics_form/state';
+import { DataFrameAnalyticsListRow } from './common';
+
+interface PropDefinition {
+  /**
+   * Indicates if the property is optional
+   */
+  optional: boolean;
+  /**
+   * Corresponding property from the form
+   */
+  formKey?: keyof State['form'];
+  /**
+   * Default value of the property
+   */
+  defaultValue?: any;
+  /**
+   * Indicates if the value has to be ignored
+   * during detecting advanced configuration
+   */
+  ignore?: boolean;
+}
+
+function isPropDefinition(a: PropDefinition | object): a is PropDefinition {
+  return a.hasOwnProperty('optional');
+}
+
+interface AnalyticsJobMetaData {
+  [key: string]: PropDefinition | AnalyticsJobMetaData;
+}
+
+/**
+ * Provides a config definition.
+ */
+const getAnalyticsJobMeta = (config: CloneDataFrameAnalyticsConfig): AnalyticsJobMetaData => ({
+  allow_lazy_start: {
+    optional: true,
+    defaultValue: false,
+  },
+  description: {
+    optional: true,
+    formKey: 'description',
+  },
+  analysis: {
+    ...(isClassificationAnalysis(config.analysis)
+      ? {
+          classification: {
+            dependent_variable: {
+              optional: false,
+              formKey: 'dependentVariable',
+            },
+            training_percent: {
+              optional: true,
+              formKey: 'trainingPercent',
+            },
+            eta: {
+              optional: true,
+            },
+            feature_bag_fraction: {
+              optional: true,
+            },
+            maximum_number_trees: {
```

Comment:
Looks like this is now `max_trees`

Reply:
fixed in https://github.com/elastic/kibana/pull/59791/commits/44cc41ba0909eac4595378a0769474c474b1f188